samples/GuestConfiguration/installed-application-linux/azurepolicyset.json

```diff
@@ -0,0 +1,30 @@
+{
+    "properties": {
+        "displayName": "Audit if an application is not installed inside Linux VMs",
```

Comment:
>an [](start = 33, length = 2)

the specified?

Reply:
Could change to "the specified", but since it takes an array of options, it could be "an application" or "applications". I guess maybe "Audit if specified applications are not installed inside Linux VMs" to be more accurate.

Reply:
Doesn't look like an array to me? It's just a single application name

---
In reply to: [263160499](https://github.com/Azure/azure-policy/pull/326#discussion_r263160499) [](ancestors = 263160499)

Reply:
It's not an "array" type in policy, but it is an "array" type inside the GuestConfig agent. Which is why the sample parameter looks like this:
`$policyParam  = '{ "installedApplication": { "value": "python,powershell" } }'`

Reply:
Updated.